sycl/test-e2e/Graph/RecordReplay/usm_memset_shortcut.cpp

```diff
@@ -0,0 +1,48 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+// Extra run to check for leaks in Level Zero using ZE_DEBUG
+// RUN: %if ext_oneapi_level_zero %{env ZE_DEBUG=4 %{run} %t.out 2>&1 | FileCheck %s --implicit-check-not=LEAK %}
```

Comment:
Hasn't that been changed to `UR_L0_DEBUG` or something?

Reply:
You're right. This test has been updated.